src/bijectors/logit.jl

```diff
@@ -26,18 +28,12 @@ up1(b::Logit{N, T}) where {N, T} = Logit{N + 1, T}(b.a, b.b)
 Base.:(==)(b1::Logit, b2::Logit) = b1.a == b2.a && b1.b == b2.b
 
 (b::Logit{0})(x::Real) = _logit(x, b.a, b.b)
-(b::Logit{0})(x) = _logit.(x, b.a, b.b)
-(b::Logit{1})(x::AbstractVector) = _logit.(x, b.a, b.b)
-(b::Logit{1})(x::AbstractMatrix) = _logit.(x, b.a, b.b)
-(b::Logit{2})(x::AbstractMatrix) = _logit.(x, b.a, b.b)
-(b::Logit{2})(x::AbstractArray{<:AbstractMatrix}) = map(b, x)
-_logit(x, a, b) = logit((x - a) / (b - a))
```

Comment:
I guess `_logit` should not be deleted?

Reply:
I mean, it depends on whether or not you want the code to work, which I wasn't planning on.


:upside_down_face: 

Nice catch, thanks!:)